images/base/files/usr/local/bin/entrypoint

```diff
@@ -232,6 +232,39 @@ enable_network_magic(){
   fi
 }
 
+configure_cgroup_root(){
+  if [ "${CGROUP_ROOT}" == "" ]; then
+    return
+  fi
+
+  echo "INFO: configuring cgroup root ${CGROUP_ROOT}"
+
+  CURRENT_CGROUP=$(grep systemd /proc/self/cgroup | cut -d: -f3)
+  CGROUP_SUBSYSTEMS=$(findmnt -lun -o source,target -t cgroup | grep "${CURRENT_CGROUP}" | awk '{print $2}')
+
+  mount --make-rprivate /sys/fs/cgroup
+
+  echo "${CGROUP_SUBSYSTEMS}" |
+  while IFS= read -r SUBSYSTEM; do
```

Comment:
this should get collapsed with the other cgroup stuff eventually, we already do a nearly identical loop.

Reply:
somewhere I have a branch with this mostly written, but I understand if this functionality is pressing.